students/Josh_HOff/lesson08/circle_class.py

```diff
@@ -0,0 +1,134 @@
+import sys
+import copy
+import math
+import pathlib
+from functools import total_ordering
+
+circles = []
+@total_ordering
+class Circle(object):
+
+    def __init__(self, value):
+        global circles
+        global templist
```

Comment:
what are these globals? should not be here

Reply:
My mistake; I removed them.